From 64dacbdf421c575f8bce00ecd0e6e848eac7a6d6 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 27 Oct 2020 00:20:37 +0200
Subject: [PATCH] [SW-12754] gaudi: security: enable hbm protection

The HBM dynamic protection has been added to U-Boot, so we can now
enable it in Preboot.
Enable the HBM protection using the HBW RR_0 and remove the TODO.

Change-Id: I57dbbde3ed7655f01b4ebdade493a03dc0f9856b
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 9b7cc1c..5a769dc 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1208,12 +1208,8 @@
 static void preboot_secure_hbw(void)
 {
 	/* HBM RR_0 0 - 32GB */
-	/*
-	 * TODO: The below is commented out temporarily until the dynamic
-	 * DRAM RR protection is in place.
-	 */
-/*	preboot_secure_hbw_rr(0, 0, 0x3FFF800000000);
-*/
+	preboot_secure_hbw_rr(0, 0, 0x3FFF800000000);
+
 	/* SRAM RR_8 0x7F_F000_0000 - 0x7F_F100_0000 (16MB) */
 	/*
 	 * TODO: The below (both SRAM parts) are commented out temporarily
